CGPs/cgp-0143.md

```diff
@@ -0,0 +1,197 @@
+- **Most Cited Institutional Data Provider in the Space**:
+Artemis Citations: https://docs.google.com/presentation/d/1Re7VbBr0Mf6nTtyO0yErE8qLOrtKqYqOjtpFC4ATYQM/edit#slide=id.g2e0f1db53f8_0_267
+- **Trusted by key Celo stakeholders already:** https://x.com/RegenRene/status/1807826195884696040 & https://x.com/MentoLabs/status/1794988382197571920
+
+# Funding Request
+Artemis offers comprehensive solutions with the Application Activity Monitor and Stablecoin Dashboard, priced based on the equivalent cost of hiring 2-3 full-time employees (FTEs) per year to build and maintain similar data infrastructure as well as what other ecosystems are paying for Artemis’ services.
+
+- **Application Activity Monitor**: $150,000 per year
+- **Stablecoin Dashboard**: $124,000 per year
+
+Our pricing model reflects the extensive expertise and ongoing support required to ensure these tools are continuously updated, reliable, and user-friendly.
+
+**Total Funding Request**: $274,000 per year
+
+**Multisig Signers**: TBD
+
+# Conclusion
+Integrating Artemis into the Celo ecosystem will significantly enhance transparency, usability, and adoption of Celo’s blockchain technology, especially towards liquid token investors and large financial institutions–the core customers of Artemis and potential investors of the Celo ecosystem. Many of these customers are not aware of all the new developments of Celo as they understand Celo as a climate focused chain (https://www.notboring.co/p/celo-building-a-regenerative-economy). Artemis can help provide investors an up-to-date, unbiased and data-backed view on the latest developments of Celo including its transition to an Ethereum L2 so that Celo remains top-of-mind in the investor community.
+
+By providing comprehensive data solutions such as the Application Activity Monitor and Stablecoin Dashboard, Artemis will also empower the Celo community with valuable insights and drive informed decision-making. This partnership aligns with Celo’s mission of financial inclusion and stability, supporting the ecosystem’s growth and long-term success.
+
+## Proposed Changes
+### Transactions
+Unique Transaction of ```634,994``` Celo tokens approval to Multisign ```0xd1a06a4477b9E4633E6E56D8f9EB9ccD39cE36Bc```
+  - Destination: ```0x471EcE3750Da237f93B8E339c536989b8978a438```
+  - Data: ```increaseAllowance``` & ```0xd1a06a4477b9E4633E6E56D8f9EB9ccD39cE36Bc```
+  - Value: ```634994000000000000000000 Celo```
+
+### Json Script
+```
+[
+    {
+      "contract": "GoldToken",
+      "address": "0x471EcE3750Da237f93B8E339c536989b8978a438",
+      "function": "increaseAllowance",
+      "args": [
+        "0xd1a06a4477b9E4633E6E56D8f9EB9ccD39cE36Bc",
+        "634994000000000000000000"
+      ],
+      "value": "0"
+    }
+]
+```
+ 
+## Verification
+This command should show a proposal with no transactions:
```

Comment:
It has a transaction, please modifiy the text for: _"A human readable version of this proposal can be found using the following command (on-chain ID determined after submission):"_

Reply:
@anthonyyim please modify text above

Reply:
Updated--let me know if it looks right @work-0xj4an